src/components/TopMenu.js

```diff
@@ -44,20 +71,27 @@ const TopMenu = () => {
         <div className="flex items-center">
           <ul className="hidden md:flex uppercase font-semibold ">
             {navItems.map(({ href, label }) => (
-              
-                <NavItem
-                  key={href}
-                  href={href}
-                  onClick={handleSmallerScreensNavigation}
-                >
-                  {label}
-                </NavItem>
-              
+
+              <NavItem
+                key={href}
+                href={href}
+                onClick={handleSmallerScreensNavigation}
+              >
+                {label}
+              </NavItem>
+
             ))}
           </ul>
           {isLoggedIn ? (
-            <div className="hidden md:flex font-semibold">
-              <span className="mr-2">{tokenDataEmail}</span>
+            <div className="relative">
+              <button onClick={handleButtonEmailActivate}>
+                <span className="mr-2">{tokenDataEmail}</span>
+              </button>
+              {butttonEmailActivate && (
```

Comment:
is really needed a flag to display the dropdown maybe you could add a sub menu and add the `cerrar sesion` item.